package.json

```diff
@@ -27,51 +27,50 @@
     "exec-sh": "^0.4.0",
     "express": "^4.18.2",
     "express-graphql": "^0.12.0",
-    "express-pino-logger": "^6.0.0",
-    "graphql": "^15.8.0",
+    "graphql": "^15.3.0",
     "graphql-bigint": "^1.0.0",
     "graphql-tag": "^2.12.6",
     "graphql-type-json": "^0.3.2",
     "js-yaml": "^4.1.0",
-    "kerberos": "^1.1.7",
+    "kerberos": "^2.0.1",
     "lodash": "^4.17.21",
     "moment": "^2.29.4",
     "mongodb": "^4.11.0",
     "node-cron": "^3.0.2",
     "pako": "^2.1.0",
-    "passport": "^0.5.3",
-    "passport-saml": "^3.2.4",
+    "passport": "^0.6.0",
```

Comment:
With this we could not login previous time.

https://github.com/jaredhanson/passport/issues/939
It was necessary to revert.
Please do not update this package to 0.6.0 until ^^^ is fixed.